src/common/input.cpp

```diff
@@ -266,6 +270,12 @@ void CPlayerInput::Update(SDL_Event event, short iGameState)
 				if (iDeviceID != event.jbutton.which)
 					continue;
 
+#ifdef __SWITCH__
+				//first player R-Trigger (button 7) enables/disables single joycon mode
+                if (iDeviceID == 0 && event.jbutton.which == 0 && event.jbutton.button == 7)
+					singleJoyconMode = !singleJoyconMode;
```

Comment:
Could you make it a setting? R trigger is a configurable and usable button in games and menu and I wouldn't want to accidentally switch mode.

Reply:
Yes I can try as soon as I get it to compile.